.github/workflows/handler.deploy-subgraph.yml

```diff
@@ -58,8 +61,16 @@ jobs:
         run: "yarn codegen"
         working-directory: ${{ env.subgraph-working-directory }}
 
-      - name: "Deploy to endpoint"
+      - name: "Deploy to Superfluid endpoint"
+        if: inputs.version_label == ""
         run: "yarn deploy ${{ github.event.inputs.release_branch }} ${{ github.event.inputs.network }}"
         working-directory: ${{ env.subgraph-working-directory }}
         env:
           THEGRAPH_ACCESS_TOKEN: ${{ secrets.THEGRAPH_ACCESS_TOKEN }}
+
+      - name: "Deploy to Satsuma endpoint"
+        if: inputs.version_label != ""
```

Comment:
This is not straightforward logic... The environment the Subgraph is deployed to is chosen based on whether you passed in an optional "version label" argument? Please make it more explicit.

Reply:
as you wish